### PR TITLE
Feat/lorena/23 spotify api fetch

### PR DESCRIPTION
**Pull request by LorenaCapraru**
- [ ] Created API_TOKEN (check confidential word doc for it)
- [ ] Installed a newer version of node as the runtime is low. just run `npm i`
- [ ] I fetched data from the API endpoint based on the id retrieved from the link entered by user
- [ ] I render the playlist details

### CODE REVIEW

**Comment by momahboobian** [comment]
Please push you pr to the `dev` branch

File: src/app/components/CheckPlaylist/components/Input/Input.tsx [hunk 1]
```diff
@@ -1,15 +1,49 @@
...
+} from "@/app/recoil/atoms";
 import { useRecoilState } from "recoil";
 
 const Input = () => {
+  const [inputSpotifyLink, setInputSpotifyLink] = useRecoilState(
+    inputSpotifyLinkState
+  );
   const [inputSpotifyId, setInputSpotifyId] =
     useRecoilState(inputSpotifyIdState);
+  const [playlistData, setPlaylistData] = useRecoilState(playlistDataState);
```
Comment (VitalinaKuzmenko) [inline src/app/components/CheckPlaylist/components/Input/Input.tsx:17]: Can you add types here (for state) as well, just to be consistent:)

**Review by VitalinaKuzmenko — APPROVED** [review]
@LorenaCapraru  Everything is great!!! Just add types in one place and you can merge!

Great job!!!👍

File: src/app/components/CheckPlaylist/components/Input/Input.tsx (hunk 1 above)
Comment (VitalinaKuzmenko) [reply src/app/components/CheckPlaylist/components/Input/Input.tsx]: I added it:)

**Pull request merged**